docs/metrics.md

```diff
@@ -57,7 +71,98 @@ Annotations required by this integration can be activated by setting the `agones
+
+- [Agones Status](../build/grafana/dashboard-status.yaml) displays Agones controller health status.
+
+- [Agones Controller Resource Usage](../build/grafana/dashboard-controller-usage.yaml) displays Agones Controller CPU and memory usage and also some Golang runtime metrics.
+
+Dashboard screenshots :
+
+![](grafana-dashboard-autoscalers.png)
+
+![](grafana-dashboard-controller.png)
+
+> You can import our dashboards by copying the json content from [each config map](../build/grafana) into your own instance of Grafana (+ > Create > Import > Or paste json) or follow the [installation](#installation) guide.
+
+## Installation
+
+When operating a live multiplayer game you will need to observe performances, resource usage and availability to learn more about your system. This guide will explain how you can setup Prometheus and Grafana into your own Kubernetes cluster to monitor your Agones workload.
+
+Before attemping this guide you should make sure you have [kubectl](https://kubernetes.io/docs/tasks/tools/install-kubectl/) and [helm](https://docs.helm.sh/using_helm/) installed and configured to reach your kubernetes cluster.
+
+### Prometheus installation
+
+Prometheus is an open source monitoring solution, we will use it to store Agones controller metrics and query back the data.
+
+Let's install Prometheus using the [helm stable](https://github.com/helm/charts/tree/master/stable/prometheus) repository.
+
+```bash
+helm install --wait --name prom stable/prometheus --namespace metrics \
+  --set pushgateway.enabled=false \
+  --set kubeStateMetrics.enabled=false,nodeExporter.enabled=false
+```
+
+> You can also run our [Makefile](../build/Makefile) target `make setup-prometheus` or `make kind-setup-prometheus` and `make minikube-setup-prometheus` for [Kind](../build/README.md#running-a-test-kind-cluster) and [Minikube](../build/README.md#running-a-test-minikube-cluster).
+
+By default we will disable the push gateway (we don't need it for Agones) and other exporters.
+
+The helm [chart](https://github.com/helm/charts/tree/master/stable/prometheus) support [nodeSelector](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#nodeselector), [affinity](https://kubernetes.io/docs/concepts/configuration/assign-pod-node/#affinity-and-anti-affinity) and [toleration](https://kubernetes.io/docs/concepts/configuration/taint-and-toleration/), you can use them to schedule prometheus deployments on an isolated node(s) to have an homogeneous game servers workload.
+
+This will install a Prometheus Server in your current cluster with [Persistent Volume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) (Desactivated for Minikube and Kind) for storing and querying time series, it will automatically start collecting metrics from Agones Controller.
```

Comment:
```suggestion
This will install a Prometheus Server in your current cluster with [Persistent Volume Claim](https://kubernetes.io/docs/concepts/storage/persistent-volumes/) (Deactivated for Minikube and Kind) for storing and querying time series, it will automatically start collecting metrics from Agones Controller.
```